src/main/java/org/isf/login/rest/LoginController.java

```diff
@@ -74,12 +78,26 @@ public class LoginController {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(LoginController.class);
 
+	@Autowired
```

Comment:
What is the purpose of this?   `@Autowired` should be removed.    SpringBoot knows what to do with this constructor.

Reply:
And all the other `@Autowired` in line 60-76 can be removed too.

Reply:
you're right, just left after some tries... for this reason maybe I had problem in testing the login failure... I will try again.
Thanks!